packages/node_modules/@ciscospark/widget-recents/README.md

```diff
@@ -0,0 +1,274 @@
+# Spark Recents Widget _(@ciscospark/widget-recents)_
```

Comment:
Yea. I eventually want to add a top level widget readme or include it in the `widget-base` docs. For now just for clarity and a single place to link the customer, I'm repeating myself a lot.